frontend/src/bundles/common/components/header-wrapper/components/avatar-header/avatar-header.tsx

```diff
@@ -0,0 +1,24 @@
+import { AppRoute } from '~/bundles/common/enums/app-route.enum.js';
+import { getUserAvatar } from '~/bundles/common/helpers/get-user-avatar';
+import { useAppSelector } from '~/bundles/common/hooks/hooks';
+
+import { Header, NavTabs } from '../../../components';
+import { UserProfile } from '../../../layout/header/user-profile/user-profile';
```

Comment:
```suggestion
import { Header, NavTabs, UserProfile } from '~/bundles/common/components/components';
```
instead of absolute path, similar code is below

Reply:
Done